$NetBSD: patch-aa,v 1.1 1998/02/01 01:48:52 tron Exp $

(This is the re.1.txt patch from www.python.org)

--- Lib/re.py	1997/12/31 21:40:43
+++ Lib/re.py	1997/12/31 21:55:49
@@ -203,6 +203,7 @@
 		if type(g)==type( "" ): g = [g]
 		results[len(results):] = list(g)
 	    pos = lastmatch = j
+	    n = n + 1
 	results.append(source[lastmatch:])
 	return results
